aikau/src/test/resources/alfresco/services/ContentServiceTest.js

```diff
@@ -99,13 +103,84 @@ define(["intern!object",
          return browser.findByCssSelector(".footer .alfresco-buttons-AlfButton:first-child .dijitButtonText")
             .click()
          .end()
-         .sleep(500)
+         .findAllByCssSelector("#ALF_DELETE_CONTENT_DIALOG.dialogHidden")
+         .end()
          .findAllByCssSelector(TestCommon.pubDataCssSelector("ALF_DISPLAY_NOTIFICATION", "message", "Delete successful"))
             .then(function(elements) {
                assert.lengthOf(elements, 2, "Could not find success notification");
             });
       },
 
+      "Edit basic metadata": function() {
+         return browser.findByCssSelector("#EDIT_BASIC_METADATA_label")
+            .click()
+         .end()
+         .findAllByCssSelector("#ALF_BASIC_METADATA_DIALOG.dialogDisplayed") // Wait for dialog
+         .end()
+         .findByCssSelector(".alfresco-forms-controls-TextBox:nth-child(2) .dijitInputContainer input")
+            .getProperty("value")
+            .then(function(text) {
+               assert.equal(text, "Some Node", "The node name was not set correctly");
+            })
+         .end()
+         .findByCssSelector(".alfresco-forms-controls-TextBox:nth-child(3) .dijitInputContainer input")
+            .getProperty("value")
+            .then(function(text) {
+               assert.equal(text, "With this title", "The node title was not set correctly");
+            })
+         .end()
+         .findByCssSelector(".alfresco-forms-controls-TextArea:nth-child(4) .dijitTextArea")
+            .getProperty("value")
+            .then(function(text) {
+               assert.equal(text, "And this description", "The node title was not set correctly");
+            })
+         .end();
+      },
+
+      "Save basic metadata": function() {
+         return browser.findByCssSelector("#ALF_BASIC_METADATA_DIALOG .confirmationButton > span")
+            .click()
+         .end()
+         .findByCssSelector("tr.mx-row:nth-child(3) .mx-url")
+            .getVisibleText()
+            .then(function(text) {
+               assert(text.indexOf("aikau/proxy/alfresco/api/node/workspace/SpacesStore/1a0b110f-1e09-4ca2-b367-fe25e4964a4e/formprocessor") !== -1, "Update POST did not include nodeRef: " + text);
```

Comment:
Would assert.include be cleaner here? (http://chaijs.com/api/assert/#include-section)